eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -941,6 +941,16 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
       goto(NORMAL) using d.copy(channelUpdate = channelUpdate) storing() replying ChannelCommandResponse.Ok
 
     case Event(BroadcastChannelUpdate(reason), d: DATA_NORMAL) =>
+      // We usually handle feerate updates once per block (~10 minutes), but when our remote is a mobile wallet that only
+      // briefly connects and then disconnects, we may never have the opportunity to send our `update_fee`, so we always
+      // send it (if needed) when reconnected.
+      if (reason == Reconnected && d.commitments.localParams.isFunder) {
+        val currentFeeratePerKw = d.commitments.localCommit.spec.feeratePerKw
+        val networkFeeratePerKw = nodeParams.onChainFeeConf.feeEstimator.getFeeratePerKw(nodeParams.onChainFeeConf.feeTargets.commitmentBlockTarget)
```

Comment:
Why not use this code in the `when(SYNCING)` handler right before going to normal?

Reply:
For the same reason that delay was added for sending the `channel_update`, to avoid flappy connections. But in the case of `update_fee` maybe it's better to send it directly at the end of the `channel_reestablish` in syncing?

Reply:
I think it's better to send it right away, because it can be viewed as maintaining the commitment tx (just like re-sending sig/rev). 

Note that https://github.com/ACINQ/eclair/pull/1371 addressed the outgoing flappy connections issue, still need to address the incoming ones (with some kind of banning probably).

Reply:
I'll move this to the end of the `channel_reestablish` handshake then.

Reply:
Done.